servicecontrol/servicecontrol-instances/configuration.md

```diff
@@ -465,7 +465,7 @@ If using [MSMQ transport](/transports/msmq) and the monitoring instance is insta
 
 | Context | Name |
 | --- | --- |
-| **Environment variable** | `LICENSECOMPONENTS_SERVICECONTROLTHROUGHPUTDADAQUEUE` |
+| **Environment variable** | `LICENSINGCOMPONENT_SERVICECONTROLTHROUGHPUTDADAQUEUE` |
```

Comment:
Still says Dada queue!

Reply:
Eagle eyes! I read that a few times and compared to what's in code and did not see it!

Reply:
Fixed in another commit